src/windows/BarcodeScannerProxy.js

```diff
@@ -17,95 +83,105 @@ module.exports = {
      * @param  {array} args       Arguments array
      */
     scan: function (success, fail, args) {
-
         var capturePreview,
             capturePreviewAlignmentMark,
             captureCancelButton,
             capture,
             reader;
-        
+
+        function updatePreviewForRotation() {
+            if (!capture) {
+                return;
+            }
+            var videoEncodingProperties = capture.videoDeviceController.getMediaStreamProperties(Windows.Media.Capture.MediaStreamType.VideoPreview);        
+
+            var rotDegree = videoPreviewRotationLookup(Windows.Graphics.Display.DisplayInformation.getForCurrentView().currentOrientation);
+
+            capturePreview.style.transform = "rotate(" + rotDegree + "deg)";
+            capturePreview.style.msTransform = "rotate(" + rotDegree + "deg)";
```

Comment:
This works fine for me only if device is rotated to landscape orientation. In portrait mode actual size of video element does not match device's screen (see attached image) 
![wp_ss_20151215_0002](https://cloud.githubusercontent.com/assets/3857604/11839646/7a8b0096-a400-11e5-9df7-a70708c6fb7a.png)